### PR TITLE
Update WinUI3 Sample with latest packages

### PR DESCRIPTION
**Pull request by Redth**

### CODE REVIEW

**Comment by jonathanpeppers** [comment]
Was this meant to go to the develop branch? It looks like it would merge develop into main.

**Comment by Redth** [comment]
Yep, waiting on some new MAUI packages before merging into develop.

File: HelloMaui/MainPage.xaml.cs [hunk 1]
```diff
@@ -20,7 +20,5 @@ private void OnButtonClicked(object sender, EventArgs e)
 			count++;
 			CounterLabel.Text = $"You clicked {count} times!";
 		}
-
-		public IView View { get => (IView)Content; set => Content = (View)value; }
```
Comment (mattleibow) [inline HelloMaui/MainPage.xaml.cs:24]: Do we still need this IPage on this?

File: HelloMauiWinUI3/HelloMauiWinUI3/HelloMauiWinUI3.csproj [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+    <RootNamespace>HelloWinUI3</RootNamespace>
+    <ApplicationManifest>Windows\app.manifest</ApplicationManifest>
+    <Platforms>x86;x64;arm64</Platforms>
+    <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
+		<EnableDefaultPageItems>False</EnableDefaultPageItems>
```
Comment (mattleibow) [inline HelloMauiWinUI3/HelloMauiWinUI3/HelloMauiWinUI3.csproj:10]: Total nit here, but I don't think the ws in this file is consistent. ;)

**Review by mattleibow — COMMENTED** [review]
Very nice! -1 project.

I think we can remove the IPage interface.

**Pull request merged**